scipy/sparse/csgraph/tests/test_flow.py

```diff
@@ -5,66 +5,76 @@
 from scipy.sparse import csr_matrix, csc_matrix
 from scipy.sparse.csgraph import maximum_flow
 
+methods = ['edmonds_karp', 'dinic']
 
 def test_raises_on_dense_input():
     with pytest.raises(TypeError):
         graph = np.array([[0, 1], [0, 0]])
         maximum_flow(graph, 0, 1)
+        maximum_flow(graph, 0, 1, method='edmonds_karp')
 
 
 def test_raises_on_csc_input():
     with pytest.raises(TypeError):
         graph = csc_matrix([[0, 1], [0, 0]])
         maximum_flow(graph, 0, 1)
+        maximum_flow(graph, 0, 1, method='edmonds_karp')
 
 
 def test_raises_on_floating_point_input():
     with pytest.raises(ValueError):
         graph = csr_matrix([[0, 1.5], [0, 0]], dtype=np.float64)
         maximum_flow(graph, 0, 1)
+        maximum_flow(graph, 0, 1, method='edmonds_karp')
 
 
 def test_raises_when_source_is_sink():
     with pytest.raises(ValueError):
         graph = csr_matrix([[0, 1], [0, 0]])
         maximum_flow(graph, 0, 0)
+        maximum_flow(graph, 0, 0, method='edmonds_karp')
```

Comment:
I think test parametriwation could be introduced here as well.
Is there a reason for not using it?

Reply:
I intentionally didn't introduce parametrization here because I want to test if default argument functionality for `maximum_flow` is working properly. These are trivial tests so thought of doing it here.

Reply:
This makes sense; thanks for precising.